gulpfile.js

```diff
@@ -323,7 +323,7 @@ gulp.task('watch', function() {
 
 
 // Start a server for serving frontend
-gulp.task('connect', function() {
+gulp.task('connect', ['lint'], function() {
```

Comment:
Any hidden purpose to add `lint` dependency to `connect` task. Since it is doing the same  as `runSequence` is doing at #L358